src/library/interface.cpp

```diff
@@ -14,16 +14,18 @@ OrbSlam2Interface::OrbSlam2Interface(const ros::NodeHandle& nh,
       nh_private_(nh_private),
       verbose_(kDefaultVerbose),
       frame_id_(kDefaultFrameId),
-      child_frame_id_(kDefaultChildFrameId) {
+      child_frame_id_(kDefaultChildFrameId){
   // Getting data and params
+  use_body_transform_ = false;
+
   advertiseTopics();
   getParametersFromRos();
 }
 
 void OrbSlam2Interface::advertiseTopics() {
   // Advertising topics
   T_pub_ = nh_private_.advertise<geometry_msgs::TransformStamped>(
-      "transform_cam", 1);
+      "orb_slam_2", 1);
```

Comment:
Why this change? The topic is publishing in the node's private namespace (for example orb_slam_2), this change will lead to a redundant and uninformative change.